requirements.txt

```diff
@@ -10,11 +10,11 @@ beautifulsoup4==4.9.3
 bleach==3.3.1
 certifi==2021.5.30
 cfenv==0.5.3
-cg-django-uaa==2.1.1
+cg-django-uaa==2.1.3
 charset-normalizer==2.0.4; python_version >= '3'
 dj-database-url==0.5.0
 django-webtest==1.9.7
-django==3.1.13
+django==3.2.6
```

Comment:
Thanks @kewlguy781! It looks like we may have updated `requirements.txt` directly. We should use pipenv to generate this from our lock file so we've got consistent definitions across both files.

```bash
pipenv lock --requirements > requirements.txt
```